### DISCUSSION TITLE
navigation jumps to station market

### DISCUSSION THREAD

**Discussion by doombreed52** [joaomlneto/jitaspace#399] [Ideas]
Hi, i just thought of a possible idea for market searches I don't live in the the best region for a market and i have to use 3rd party apps to search outside of my region but I don't know how many jumps it is to the station I'm looking at without a lot of copy and pasting, would it be possible to add a tab or column of some kind that gives me jumps from a starting location to see what the closest station is that has something I'm looking for.

**Comment by joaomlneto** [comment]
That could certainly be done, but there's so many things I need to fix :) things are still a bit experimental before I add that kind of QOL improvements. Do feel free to contribute it if you feel! I might throw some PLEX your way :)
